pandas/core/describe.py

```diff
@@ -181,7 +181,9 @@ def describe(self, percentiles: Sequence[float]) -> DataFrame:
             sort=False,
         )
         d.columns = data.columns.copy()
-        return d
+        # Incompatible return value type (got "Union[DataFrame, Series]",
+        # expected "DataFrame")
+        return d  # type: ignore[return-value]
```

Comment:
this goes away once concat has overloads?

Reply:
Yup!